Bottom: 43573e33946f0f5161e3660757d58512f148b4a0
Top:    97e8b521eeff84d3264fb4dcbfd9ef38e02d0a7b
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-05-11 15:34:04 +0800

Refresh of vhost_net-add-interrupt

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 7d137a4..80d9f6c 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -415,6 +415,7 @@ static void handle_tx(struct vhost_net *net)
 		}
 	}
 out:
+	vhost_check_coalesce_and_signal(dev, vq);
 	mutex_unlock(&vq->mutex);
 }
